snap/info.go

```diff
@@ -987,3 +995,28 @@ func DropNick(nick string) string {
 	}
 	return nick
 }
+
+// StoreName splits the maybe-local name and returns the store name of the snap.
+func StoreName(name string) string {
+	return strings.SplitN(name, "_", 2)[0]
+}
+
+// SplitName splits the maybe-local name and returns the store name and the
+// local key.
+func SplitName(name string) (store string, localKey string) {
+	split := strings.SplitN(name, "_", 2)
+	store = split[0]
+	if len(split) > 1 {
+		localKey = split[1]
+	}
+	return store, localKey
+}
+
+// LocalName takes the store name and the local key and returns a local name of
+// the snap.
+func LocalName(storeName string, localKey string) string {
```

Comment:
Can we call it instance key instead of a local key? Local key tells us nothing concrete. Unless there are non-instance uses that we are planning I would just be concrete/direct everywhere.